domains/ho2.json

```diff
@@ -0,0 +1,12 @@
+{
+    "owner": {
+        "username": "ho2",
+        "email": "hussainaltahoo944@gmail.com",
+        "repo": "https://ho2.vercel.app/",
+        "description": "ho2 project"
+    },
+    "record": {
+        
+        "TXT": "vc-domain-verify=ho2.is-a.dev,e925e90a6f992a727337"
```

Comment:
we do not support vercel (assuming since vc-domain-verify is a vercel thing when googled)